lib/core/src/server/config/useBaseTsSupport.js

```diff
@@ -0,0 +1,8 @@
+/**
+ * Returns true if the framework can use the base TS config.
+ * @param {string} framework
+ */
+export const useBaseTsSupport = (framework) => {
```

Comment:
TS => Typescript ?

Reply:
Correct :) I could rename if you think it's unclear?